packages/panda-plugins/src/strict-tokens-scope.ts

```diff
@@ -10,6 +10,7 @@ import type { PandaContext } from '@pandacss/node'
 export interface StrictTokensScopeOptions {
   categories?: TokenCategory[]
   props?: Array<keyof CssProperties | (string & {})>
+  shouldExclude?: boolean
```

Comment:
I think the `excludeCategories: ['colors', 'spacing']` made more sense here instead of a boolean

Reply:
I fix it.